sdk-setup/src/mb2

```diff
@@ -2070,6 +2075,24 @@ run_package() {
             -bb --noprep --nobuildstage --noclean \
             "$TMP_SPEC"
     )
+    [[ -z $NO_SIGN_RPM ]] && run_sign
+}
+
+run_sign() {
```

Comment:
I would prefer not using run_ prefix here - this is used for functions implementing subcommands. I would call it sign_rpms and move its definition closer to the beginning of the file where other reusable functions are defined.